src/components/shared/Button/Button.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './Button.scss';
+
+const Button = ({ children, ...otherProps }) => {
+  return (
+    <button className='button effect' {...otherProps}>
+      <span className='button__span'>{children}</span>
```

Comment:
klasa button jest ok jako ogolna ale juz modifier to powinno byc raczej button--effect ale przemyslalbym tutaj te propsy bardziej zeby mozna bylo przekazac moze. tez kolor buttona ktory automatycznie bedzie bindowany do classy poza tym ta klasa span nic nie mowi bardziej button__text.